dataflow-samples/bert-qa-trt-dataflow/README.md

```diff
@@ -0,0 +1,37 @@
+# TensorRT Bert Q&A Inference in GCP Dataflow
+
+## Sample Overview
+
+This is sample that deployed a optimized BERT large model with sequence-length 384, fine-tuned with squad Q&A task, then run inference with GCP dataflow. To run the sample, please first create a docker image and push to GCR registry with [build_and_push.sh](build_and_push.sh). Please make sure your git lfs is enabled so that you can download the engine file. In the sample code, we generate replicated documentation and questions to send, each request has a default batch size of 16 which match the optimial batch size of the TensorRT Engine profile. 
+
+Once image is pushed successfully, make sure your local python environment has version 3.6 and properly installed with dataflow prerequsite, then dataflow job could be launch with:
```

Comment:
Add steps needed for user to launch from local environment
`pip install apache-beam`

Reply:
added getting started guide